fashrevwall/test_twitter_client.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
But they probably need to be done the django way, which I'm still not sure what it is! :)